### PR TITLE
ST: add support for set specific remote cluster and execute tests there

### PR DESCRIPTION
**Pull request by Frawless**
### Type of change

- Enhancement / new feature

### Description

Option for add remote cluster to system tests. Problem is, that env variable `KUBERNETES_API_TOKEN` has to be set even for local cluster. 

### Checklist

- [ ] Make sure all tests pass
- [ ] Update documentation
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: systemtest/scripts/run_tests_kubernetes.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+CURDIR=`readlink -f \`dirname $0\``
+source ${CURDIR}/test_func.sh
+
+TESTCASE=${1:-.*ST}
+JUNIT_TAGS=${2:-acceptance}
+TEST_PROFILE=${3:-systemtests}
+DEFAULT_KUBERNETES_NAMESPACE="myproject"
```
Comment (serrss) [inline systemtest/scripts/run_tests_kubernetes.sh]: I guess default  Kubernetes namespace is `default`, not `myproject`.

**Comment by Frawless** [comment]
@strimzi-ci run tests

File: .travis/build.sh [hunk 2]
```diff
@@ -19,6 +19,12 @@ export DOCKER_ORG=${DOCKER_ORG:-strimzici}
 export DOCKER_REGISTRY=${DOCKER_REGISTRY:-docker.io}
 export DOCKER_TAG=$COMMIT
 
+API_URL=$(kubectl config view --minify | grep server | cut -f 2- -d ":" | tr -d " ")
+API_TOKEN=$(kubectl describe secret $(kubectl get serviceaccount default -o jsonpath='{.secrets[0].name}') | grep -E '^token' | cut -f2 -d':' | tr -d " ")
```
Comment (scholzj) [inline .travis/build.sh]: The default service account should normally have minimal privileges. I'm not sure I understand wat is the value of getting its token. Moreover AFAIK not every service account can actually authenticate from the outside. You anyway seem to redefine these things later, so I'm not sure what is the value of this change in `build.sh`.

**Comment by Frawless** [comment]
@scholzj good point. I update code so currently it will load specific cluster context - if `TEST_CLUSTER_CONTEXT` will be set, it will load context with name in this variable, if this env var will be empty, it will use currently active context. However, since we are using executor for many actions, context for remote cluster have to be active during execution. WDYT?

**Comment by scholzj** [comment]
> However, since we are using executor for many actions, context for remote cluster have to be active during execution. WDYT?

You mean that you will change the active context for the System Test run? That sounds fine to me.

**Comment by Frawless** [comment]
@strimzi-ci run tests

**Pull request merged**